### ISSUE TITLE
Fix CRAN error

### ISSUE THREAD

**Issue by peterdesmet** [frictionlessdata/frictionless-r#113]
CRAN currently shows an ERROR: https://cran.r-project.org/web/checks/check_results_frictionless.html

This is caused by a URL timeout in one of the tests (on one OS only, r-devel-linux-x86_64-fedora-gcc). It was commented we should more safely code for that.

```
Error in `open.connection(con, "rb")`: Timeout was reached: [github.com] SSL connection timeout
     Backtrace:
     ▆
     1. └─frictionless::add_resource(p, "remote", remote_path, schema) at test-add_resource.R:257:2
     2. └─readr::guess_encoding(last_file, n_max = 1000)
     3. ├─base::unlist(read_lines_raw(file, n_max = n_max))
     4. └─readr::read_lines_raw(file, n_max = n_max)
     5. └─readr::datasource(...)
     6. └─readr:::datasource_connection(...)
     7. └─readr:::read_connection(path)
     8. ├─base::open(con, "rb")
     9. └─base::open.connection(con, "rb")
```

**Comment by damianooldoni** [comment]
Yes, I read a lot months ago about mocks and cassettes and vcr package. I think I applied to some of our packages (I have to remember where I applied it).

However, I think you could also safely apply the `skip_on_cran()` function in your tests. 
See an example from rgbif package: https://github.com/ropensci/rgbif/blob/master/tests/testthat/test-downloads.R#L3

**Comment by damianooldoni** [comment]
Now I remember where I applied mocks: it is for testing the `inborutils::setup_coding_club_session()`. See https://github.com/inbo/inborutils/blob/14729155612aff576e554d5091d8660547e2b2d0/tests/testthat/test-setup_codingclub_session.R#L23-L28 where I use `stub()` from the `mockery` package to "stub" the function and simulate some R basic functions.

**Comment by peterdesmet** [comment]
Ok, need to fix this asap, package was removed: https://cran.r-project.org/web/packages/frictionless/index.html

**Comment by peterdesmet** [comment]
Resolved in #116.
